src/Stoa.ts

```diff
@@ -44,17 +45,23 @@ class Stoa extends WebService
      */
     private storage_filename: string;
 
+    /**
+     * Main config
+     */
+    private config: Config;
+
     /**
      * Constructor
      * @param database_filename sqlite3 database file name
      * @param agora_endpoint The network endpoint to connect to Agora
      * @param port The network port of Stoa
      * @param address The network address of Stoa
      */
-    constructor (database_filename: string, agora_endpoint: URL, port: number | string, address: string)
+    constructor (config: Config, database_filename: string, agora_endpoint: URL, port: number | string, address: string)
```

Comment:
We did not use Config directly inside Stoa to avoid creating a dependency between Class Config and Class Stoa.
Therefore, it would be a little uncomfortable, but it would be better to deliver only the necessary variables.
```suggestion
    constructor (database_filename: string, agora_endpoint: URL, port: number | string, address: string, genesis_timestamp: number)
```